docker-compose.yml

```diff
@@ -26,7 +26,6 @@ services:
       - ./app/:/app:rw
   db:
     image: mysql/mysql-server
-    container_name: myPHPexperiments_MySQL
```

Comment:
just a minor cleanup -- normally you dont need to wset container names in docker-compose, just use the defuaults